phpunit.xml.dist

```diff
@@ -32,5 +32,6 @@
         <env name="KERNEL_DIR" value="./tests/Functional/Fixtures/app" />
         <env name="KERNEL_CLASS" value="AppKernel" />
         <env name="SYMFONY_DEPRECATIONS_HELPER" value="weak" />
+        <server name="SYMFONY_PHPUNIT_VERSION" value="6.5" />
```

Comment:
i think this is breaking some tests `Call to undefined method FOS\HttpCacheBundle\Tests\Unit\Command\ClearCommandTest::assertStringContainsString()`

Reply:
oh, actually those all come from the test you add, so the version of phpunit we use might simply not provide the StringContainsString assertion.